app/services/importer/fields_builder.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Importer
+  class FieldsBuilder < ApplicationTask
+    def initialize(data)
+      @data = data
+    end
+
+    def call
+      return [] unless processable
+
+      process
+    end
+
+    private
+
+    def processable
+      false
+    end
+
+    def process
+      # All yours, JB!
```

Comment:
FAO @patanj101. Think this is a repeated file at the moment no @daniel-sussman?

Reply:
We work on 2 different branches. Dan has probably not seen mine ... it will be merge and mine will take over.
All good.